tikv-configuration-file.md

```diff
@@ -693,6 +693,10 @@ Configuration items related to Raftstore.
 
 ### `consistency-check-interval`
 
+> **Warning:**
+>
+> It is **NOT** recommended to enable consistency check in production environment, because it will affect the performance of the cluster and is not compatible with the TiDB Garbage Collection.
```

Comment:
```suggestion
> It is **NOT** recommended to enable the consistency check in production environments, because it affects cluster performance and is incompatible with the garbage collection in TiDB.
```